mapreader/annotate/utils.py

```diff
@@ -664,31 +659,32 @@ def save_annotation(
 
     # Read an existing annotation file (for the same task and userID)
     try:
-        image_df = pd.read_csv(annot_file)
+        image_df = pd.read_csv(annot_file, sep="\t", index_col=0)
     except:
-        image_df = pd.DataFrame(columns=["image_id", "label"])
+        image_df = pd.DataFrame(columns=["image_id", "image_path", "label"])
 
     new_labels = 0
     newly_annotated = 0
     for i in range(len(annotation.tasks)):
         if annotation.tasks[i].value is not None:
             newly_annotated += 1
             if (
-                not annotation.tasks[i].output[0]
-                in image_df["image_id"].values.tolist()
+                annotation.tasks[i].output[0]
+                not in image_df["image_id"].values.tolist()
             ):
                 image_df = image_df.append(
                     {
                         "image_id": annotation.tasks[i].output[0],
+                        "image_path": annotation.tasks[i].output[1],
                         "label": annotation.tasks[i].value,
                     },
                     ignore_index=True,
                 )
                 new_labels += 1
 
     if len(image_df) > 0:
-        image_df = image_df.set_index("image_id")
-        image_df.to_csv(annot_file, mode="w")
+        #image_df = image_df.set_index("image_id")
+        image_df.to_csv(annot_file, mode="w", sep="\t")
         print(f"[INFO] Save {newly_annotated} new annotations to {annot_file}")
         print(f"[INFO] {new_labels} labels were not already stored")
         print(f"[INFO] Total number of annotations: {len(image_df)}")
```

Comment:
"**Saved** {newly_annotated} new annotations to {annot_file}

"Total number of annotations" --> "Total number of **saved** annotations"